### PR TITLE
Changed media from only being images to more types

### PR DESCRIPTION
**Pull request by logan-anderson**

### CODE REVIEW

**Review by perkinsjr — DISMISSED** [review]
LGTM

**Comment by logan-anderson** [comment]
@perkinsjr I removed some console.logs but it should be good to go again.

**Review by perkinsjr — APPROVED** [review]

File: packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export class CloudinaryMediaStore implements MediaStore {
   fetchFunction = (input: RequestInfo, init?: RequestInit) => {
     return fetch(input, init)
   }
-  accept = '*'
+  accept = 'text/*,  application/*, image/*'
```
Comment (DirtyF) [inline packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts:28]: what does it mean concretely for Cloudinary users, what type of files are now supported?

Comment (Enigmatical) [reply packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts:28]: These are MIME types:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

I couldn't find a list of _just_ `text`, `application`, or `image` types, but if you go through that list, you'll see the `MIME Type` column.  If the `MIME Type` begins with `text/`, `application/` or `image/`, we'll accept it.

I'm not quite sure there is an exhaustive list (especially for `application/` where that could be any application's file type).

Comment (DirtyF) [reply packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts:28]: So we accept things like zip files now, should we be concerned about security?

Comment (mystrdat) [reply packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts:28]: Could we get support for `audio/` MIME type?

**Pull request merged**